liquid-rust-syntax/src/ast.rs

```diff
@@ -37,6 +37,14 @@ pub struct FnSig {
     pub span: Span,
 }
 
+#[derive(Debug)]
+pub struct Qualifier {
+    pub name: Ident,
+    pub args: Vec<(Ident, Ident)>,
```

Comment:
Define a struct for `(Ident, Ident)` there's no way to know which one is which by just looking at the types. Maybe you can call it `QualifParam` or something